run/GCClassic/createRunDir.sh

```diff
@@ -456,8 +456,13 @@ if [[ ${met} = "ModelE2.1" ]]; then
     # NOTE: Benchmark simulations always use the climatological emissions!
     if [[ "x${sim_name}" == "xfullchem" ]]  ||  \
        [[ "x${sim_name}" == "xaerosol"  ]]; then
-        RUNDIR_VARS+="RUNDIR_VOLC_CLIMATOLOGY='\$ROOT/VOLCANO/v2021-09/so2_volcanic_emissions_CARN_v202005.degassing_only.rc'\n"
-        RUNDIR_VARS+="RUNDIR_VOLC_TABLE='\$ROOT/VOLCANO/v2021-09/so2_volcanic_emissions_CARN_v202005.degassing_only.rc'\n"
+        RUNDIR_VARS+="RUNDIR_VOLC_CLIMATOLOGY='\$ROOT/VOLCANO/v2024-03/so2_volcanic_emissions_CARN_v202302.degassing_only.rc'\n"
+
```

Comment:
@msulprizio: The degassing_only volcano file in the `HEMCO/VOLCANO/v2024-03/` folder has version `v202106` and not `v202302`.
```console
[holy8a24401 v2024-03]$ ls -l
total 64
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 10:36 2010/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 10:47 2011/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 10:56 2012/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:03 2013/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:10 2014/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:17 2015/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:27 2016/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:38 2017/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:47 2018/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 11:55 2019/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 12:05 2020/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 12:13 2021/
drwxrwsr-x. 14 msulprizio jacob_gcst 4096 2024-03-14 12:19 2022/
-rw-rw-r--.  1 msulprizio jacob_gcst 4330 2024-03-14 10:23 README
-rw-rw-r--.  1 msulprizio jacob_gcst 3959 2024-03-12 13:52 so2_volcanic_emissions_CARN_v202106.degassing_only.rc
```

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/2198/commits/bb993b2f352ac900b63b94239c26473e7da0a25c